GoogleForCreators/web-stories-wp
# Media: Better prepend uploading items to library

## Context



## Summary



Changes the way in-progress (uploading) items are displayed in the media library to ensure they're never lost when e.g. filtering the library afterwards.

## Relevant Technical Choices



## To-do



## User-facing changes



Uploading items are still being displayed, even when changing filters in the library:

https://user-images.githubusercontent.com/841956/185426575-ef16adb3-7aff-49ae-a9b3-8932b9e10b69.mov


## Testing Instructions






This PR can be tested by following these steps:

1. Upload media to the media library
2. See uploading items in the library
3. Filter media library by type
4. Still see uploading items in the library
5. Verify the uploaded items remain in library after upload finishes


## Reviews

### Does this PR have a security-related impact?



No

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #9964
